lib/api-request.js

```diff
@@ -10,4 +11,12 @@ module.exports.get =  function() {
         port: PORT,
         scheme: SCHEME
     });
-}
\ No newline at end of file
+}
+
+module.exports.getHandler = function() {
```

Comment:
I think it's better to move this logic to another file, for example `handler-request.js`, similar to `api-request.js` file, and then use it like this:

```js
var HendlerRequest = require('./handler-request')

var request = HendlerRequest.get()
    .setPath('/handlers/artist.jsx')
    // ...
```